internal/pkg/instrumentation/bpf/database/sql/probe.go

```diff
@@ -8,6 +8,8 @@ import (
 	"os"
 	"strconv"
 
+	sql "github.com/xwb1989/sqlparser"
```

Comment:
I looked through a couple sql parsing libraries, this isn't the most maintained... https://github.com/krasun/gosqlparser seems good but for some reason I don't think it supports `*` in queries (??). For what we're doing now I think it's fine but this could use another look some day